bin/assets/scripts/newform.js

```diff
@@ -6,15 +6,13 @@ code.addEventListener('keydown', (event) => {
   if (event.code === 'Tab') {
     // prevents tab from being used to navigate between browser elements
     event.preventDefault();
-
-    const pos = code.selectionStart;
-    const content = code.value;
+    const { value: content, selectionStart, selectionEnd } = code;
 
     // inserts tab at the position of the caret
-    code.value = content.substring(0, pos) + '\t' + content.substring(pos);
+    code.value = `${content.slice(0, selectionStart)}\t${content.slice(selectionEnd)}`;
```

Comment:
```suggestion
    code.value = `${value.slice(0, selectionStart)}\t${value.slice(selectionEnd)}`;
```
*suite.*
(j'ai pas vu faire en une seule suggestion, vu que je suis sur téléphone.)